### ISSUE TITLE
Anyway to disable Joining?

### ISSUE THREAD

**Issue by Dirwinc** [ozmartian/vidcutter#94]
This is probably not a bug, but didn't know where else to ask this question.   Love VidCutter, but wish I could disable the Joining at the end of the "Save Media" button action.  Am I missing how to disable it, or is always Joining a hard-coded feature of VidCutter?  All I want to do is clip a really long vid into multiple clips, and don't join it. Thanks.

**Comment by PoulKalff** [comment]
I don't know why you'd want that, but you can add an extra clip, and while that one is processing, copy the cuts/individual files that have already been made.... I did that to test another issue (that each cut is adding a few seconds before the cut)

**Comment by Dirwinc** [comment]
Ok, that's not a bad idea.  Thanks for the suggestion. 

I have very long MP4 files (up to 6 hours a piece) that were captured from old family VHS tapes that have random and non-linear family events, so it doesn't make sense just to cut out short bits.  Some of these files I need to chop down into 20-30 clips to make it easier to share on social media (afterall, who wants 6 hours of video on a Facebook post).  Due to the amount of clips and length of time it will take to cut them all once cutting begins, it will be hard to be there at the computer right when the cutting is finishing up, so it'd be nice if there were an option to disable joining.  

If there's a way to submit a feature request, please let me know and I'd be happy to do so.  Thanks.

**Comment by PoulKalff** [comment]
Ok, that makes sense. Also, I guess since this is Python, it would be pretty straightforward to go through the code and comment out the part that deletes the files after joining them. Probably just one '#' carefully placed....

**Comment by Dirwinc** [comment]
Yeah, that's a good point.  I was a coder in a past life, so I might give that a try.  Thanks!

**Comment by kn00tcn** [comment]
if you have the cut points noted, it might be easier to do it with ffmpeg
